lib/reporter/cli_reporter.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-const notDeepStrictEqual = require('assert').notDeepStrictEqual;
 const chalk = require('chalk');
 const cliui = require('cliui');
+const Reporter = require('./reporter');
+
```

Comment:
Another nit-pick that would've been caught by code linting - can we remove the extra empty line here? Thanks.